query_execution/Worker.cpp

```diff
@@ -94,4 +75,27 @@ void Worker::run() {
   }
 }
 
+void Worker::sendWorkOrderCompleteMessage(const tmb::client_id receiver,
+                                          const size_t op_index,
+                                          const bool isRebuildWorkOrder) {
```

Comment:
The style of variable naming is different than our usual standard which is - ``is_rebuild_workorder``